addon/i3dio/ui/object.py

```diff
@@ -240,21 +342,56 @@ def draw(self, context):
                 row_compound.enabled = False
                 obj.i3d_attributes.property_unset('compound')
 
-            row = layout.row()
-            row.prop(obj.i3d_attributes, 'collision')
+            layout.prop(obj.i3d_attributes, 'collision')
+            layout.prop(obj.i3d_attributes, 'collision_mask')
+            layout.prop(obj.i3d_attributes, 'trigger')
+            layout.prop(obj.i3d_attributes, 'restitution')
+            layout.prop(obj.i3d_attributes, 'static_friction')
+            layout.prop(obj.i3d_attributes, 'dynamic_friction')
+            layout.prop(obj.i3d_attributes, 'linear_damping')
+            layout.prop(obj.i3d_attributes, 'angular_damping')
+            layout.prop(obj.i3d_attributes, 'density')
+
+            row_split_type_presets = layout.row()
+            row_split_type_presets.prop(obj.i3d_attributes, 'split_type_presets')
+
+            row_split_type = layout.row()
+            row_split_type.prop(obj.i3d_attributes, 'split_type')
+            if obj.i3d_attributes.split_type_presets != '0':
+                row_split_type.enabled = False
+
+            row_split_uvs = layout.row()
+            row_split_uvs.prop(obj.i3d_attributes, 'split_uvs')
+
+            if obj.i3d_attributes.rigid_body_type != 'static':
+                row_split_type.enabled = False
+                row_split_uvs.enabled = False
+                row_split_type_presets.enabled = False
+                obj.i3d_attributes.property_unset('split_type')
+                obj.i3d_attributes.property_unset('split_type_presets')
+                obj.i3d_attributes.property_unset('split_uvs')
+            else:
+                if obj.i3d_attributes.split_type == '0':

```

Comment:
Under what conditions does this need to be disabled? Is it when the split type is "0" as in None? Also you are comparing an integer with the character zero and that does not work.

Reply:
Just me that forgot to change it from string to int.

When split type is set to 0 in the UI it should not be possible to edit the split_uvs and therefor also reset to defaults.